tests/contrib/flask/test_flask_appsec.py

```diff
@@ -268,12 +269,18 @@ def test_flask_body_xml_empty_logs_warning(self):
     def test_flask_ipblock_match_403_json(self):
         with override_global_config(dict(_appsec_enabled=True)), override_env(dict(DD_APPSEC_RULES=RULES_GOOD_PATH)):
```

Comment:
May not apply to this PR, but do we have any tests that make sure we're not assigning tags and blocking when we shouldn't be?

Reply:
Yep: `test_flask_userblock_match_200_json`, same on Django and Django snapshots, but I'll add a non blocking test on this file too.